doc/api/crypto.markdown

```diff
@@ -666,8 +666,8 @@ The `private_key` argument can be an object or a string. If `private_key` is a
 string, it is treated as a raw key with no passphrase. If `private_key` is an
 object, it is interpreted as a hash containing two properties:
 
-* `key` : A string holding the PEM encoded private key
-* `passphrase` : A string of passphrase for the private key
+* `key` : {String} - PEM encoded private key
```

Comment:
@thefourtheye I agree, but unfortunately everything was already inconsistent. I think refactoring that belongs to another PR IMHO